dashboard/test/models/concerns/user/age_test.rb

```diff
@@ -0,0 +1,177 @@
+require 'test_helper'
+
+class AgeTest < ActiveSupport::TestCase
+  describe 'validations' do
+    context 'when the user is not sponsored' do
+      let(:user) {create(:user, age: 10)}
+      it 'validates age presence on create' do
+        _(user).must_be :valid?
+      end
+    end
+    context 'when the user has a google_oauth2 provider' do
+      let(:user) {create(:user, :google_sso_provider, age: nil)}
```

Comment:
Interested to think what @artem-vavilov and @Hamms thinks about this.
I've been using the no-parenthesis style when creating factories like this: `let(:user) {create :user, :google_sso_provider, age: nil}`
I don't really recall why or what example I was following, but there are [linting rules](https://docs.rubocop.org/rubocop-rspec/2.31/cops_rspec_factorybot.html#enforcedstyle-omit_parentheses) to enforce consistency.
I'd like to choose one rule and enforce it.

Reply:
I personally prefer parenthesis; I agree going without is more ruby-ish, but I dislike ruby's preference for implicit over explicit. But I think consistency is valuable, so I'd be in favor of enabling a linting rule even if the option we choose for it is not the one I'd personally prefer :+1:

Reply:
I have been doing some research on this, and there is some cognitive benefit from omitting unnecessary parentheses.
Moreover, Ruby style guide prefers to [omit decorators on DSL Method calls](https://rubystyle.guide/#no-dsl-decorating).

Reply:
Note that it also prefers to [use decorators for method calls by default](https://rubystyle.guide/#method-invocation-parens)

I think in this case, `let` is part of a DSL but `create` is a helper method provided by FactoryBot